packages/@lwc/compiler/README.md

```diff
@@ -42,6 +42,7 @@ const { code } = await transform(source, filename, options);
 
 -   `code` (string) - the compiled source code.
 -   `map` (object) - the generated source map.
+-   `warnings` (array, optional) - the array of diagnostic warnings, if any.
```

Comment:
Why not return all the diagnostics instead of just the warnings and the consumer decide what to do with the diagnostics?

Reply:
Looking at the oss compiler, all `Error` diagnostics cause an exception. There are no `Log` and `Fatal` level Diagnostics. That effectively leaves only `Warning` level diagnostics that can be present in this output.

Reply:
Correct. I can add a comment to this effect.